docs/docsite/rst/user_guide/become.rst

```diff
@@ -73,7 +73,7 @@ ansible_become_method
 ansible_become_user
     set the user you become through privilege escalation; does not imply ``ansible_become: yes``
 
-ansible_become_pass
+ansible_become_password
```

Comment:
I don't think this is correct, there is work to introduce become plugins where you can set variables in the config like connection options but right now this is magic done in the play context.

Reply:
At the core meeting today, it was agreed to change all `ansible_*_pass` vars to `ansible_*_password` and all `ansible_*_username` vars to `ansible_*_user`.  This particular change is just a docs change.  You already can use `ansible_become_password` or `ansible_become_pass`, though I'm unsure of which will win today if you specify both.

Reply:
Ah ok, I see https://github.com/ansible/ansible/blob/devel/lib/ansible/constants.py#L166 is where the current magic happens. I wasn't aware that `ansible_become_password` worked today but good to know.